docs/source/architecture/contracts.md

```diff
@@ -82,9 +82,15 @@ The first parameter is used to identify the stake to divide and the last two for
 When calculating locked tokens using the `StakingEscrow.getLockedTokens(address, uint16)` method, all stakes that are active during the specified period are summed.
 
 
+### The Staker Bonds to a Worker ("Ursula")
+The staker must specify a worker who will confirm the activity and sign on behalf of this staker by calling the `StakingEscrow.setWorker(address)` method.
```

Comment:
Should it be backquotes or double-backquotes?

Reply:
Single quotes in Markdown, double quotes in RST.

Reply:
Ah true, that's markdown...